autoware_adapi_v1_msgs/CMakeLists.txt

```
@@ -22,6 +22,8 @@ rosidl_generate_interfaces(${PROJECT_NAME}
   planning/msg/SteeringFactorArray.msg
   planning/msg/VelocityFactor.msg
   planning/msg/VelocityFactorArray.msg
+  system/msg/MRMState.msg
+  system/srv/OperateMRM.srv
```

https://github.com/orgs/autowarefoundation/discussions/2824#discussioncomment-3911938

```suggestion
  system/msg/MrmState.msg
  system/srv/OperateMrm.srv
```